.github/workflows/nextjs_bundle_analysis.yml

```
@@ -0,0 +1,117 @@
+           with:
+              # if you use a custom build directory, replace all instances of `.next` in this file with your build directory
+              # ex: if your app builds to `dist`, replace `.next` with `dist`
+              path: .next/cache
+              # change this if you prefer a more strict cache
+              key: ${{ runner.os }}-build-${{ env.cache-name }}
+
+         - name: Build next.js app
+           # change this if your site requires a custom build command
+           run: ./node_modules/.bin/next build
+           env:
+              ALGOLIA_API_KEY: ${{ secrets.ALGOLIA_API_KEY }}
+              NEXT_PUBLIC_STRAPI_ORIGIN: https://main.govinor.com
+
+         # Here's the first place where next-bundle-analysis' own script is used
+         # This step pulls the raw bundle stats for the current bundle
+         - name: Analyze bundle
+           run: npx -p nextjs-bundle-analysis report
+
+         - name: Upload bundle
+           uses: actions/upload-artifact@v2
+           with:
+              name: bundle
+              path: .next/analyze/__bundle_analysis.json
+
+         - name: Download base branch bundle stats
+           uses: dawidd6/action-download-artifact@v2
+           if: success() && github.event.number
+           with:
+              workflow: nextjs_bundle_analysis.yml
+              branch: ${{ github.event.pull_request.base.ref }}
+              path: .next/analyze/base
+
+         # And here's the second place - this runs after we have both the current and
+         # base branch bundle stats, and will compare them to determine what changed.
+         # There are two configurable arguments that come from package.json:
+         #
+         # - budget: optional, set a budget (bytes) against which size changes are measured
+         #           it's set to 350kb here by default, as informed by the following piece:
+         #           https://infrequently.org/2021/03/the-performance-inequality-gap/
+         #
+         # - red-status-percentage: sets the percent size increase where you get a red
+         #                          status indicator, defaults to 20%
+         #
+         # Either of these arguments can be changed or removed by editing the `nextBundleAnalysis`
+         # entry in your package.json file.
+         - name: Compare with base branch bundle
+           if: success() && github.event.number
+           run: ls -laR .next/analyze/base && npx -p nextjs-bundle-analysis compare
+
+         - name: Get comment body
+           id: get-comment-body
+           if: success() && github.event.number
+           run: |
+              body=$(cat .next/analyze/__bundle_analysis_comment.txt)
+              body="${body//'%'/'%25'}"
+              body="${body//$'\n'/'%0A'}"
+              body="${body//$'\r'/'%0D'}"
+              echo ::set-output name=body::$body
+
+         - name: Find Comment
+           uses: peter-evans/find-comment@v1
```

I think this template is a bit outdated, we currently use `v2` for this action in `iFixit/ifixit` https://github.com/iFixit/ifixit/blob/master/.github/actions/issue-comment/action.yml#L16

```suggestion
           uses: peter-evans/find-comment@v2
```